File: src/main/java/com/fwdekker/randomness/word/WordSettingsAction.java
package com.fwdekker.randomness.word;

import com.fwdekker.randomness.SettingsAction;
import com.fwdekker.randomness.SettingsDialog;


/**
 * Controller for random string generation settings.
 */
public final class WordSettingsAction extends SettingsAction {
    @Override
    public SettingsDialog createDialog() {
        return new WordSettingsDialog();
    }

    @Override
    public String getTitle() {
        return "Word Settings";
    }
}

File: src/main/kotlin/com/fwdekker/randomness/word/WordSettingsAction.kt
package com.fwdekker.randomness.word

import com.fwdekker.randomness.SettingsAction


/**
 * Controller for random string generation settings.
 */
class WordSettingsAction : SettingsAction() {
    override val title = "Word Settings"


    public override fun createDialog() = WordSettingsDialog()
}
